src/Factories/RoundFactory.php

```diff
@@ -21,17 +21,27 @@ class RoundFactory
      * @param int $playersPerRace
      * @param bool $mixedAlignment
      * @return Round
+     * @throws \Exception
      */
     public function create(RoundLeague $league, Carbon $startDate, int $realmSize, int $packSize, int $playersPerRace, bool $mixedAlignment): Round
     {
         $number = ($this->getLastRoundNumber($league) + 1);
+        $endDate = (clone $startDate)->addDays(static::ROUND_DURATION_IN_DAYS);
+
+        $invasionEndHours = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 12, 13, 14, 15, 16, 12, 13, 14, 15, 16];
+        shuffle($invasionEndHours);
+
+        $hoursBeforeRoundEnd = $invasionEndHours[random_int(0, count($invasionEndHours) - 1)];
```

Comment:
Please use Laravel's array helper instead: https://laravel.com/docs/5.7/helpers#method-array-random

Try `array_random($array)`. If that doesn't exist in the global namespace then use `Arr:random($array)`.